html_app/ui/FacsView.js

```diff
@@ -69,31 +69,50 @@ scb.ui.static.FacsView.scb_f_facs_cell_treatment_radio = function(element, event
     alert("INVALID ELEMENT!");
   }
   /* val is 'fixed' or 'live' */
-  var val = $(element).val();
+  var live = $(element).val();
   /* if lane does not exist equals to '' */
   var lane_id = $(element).attr('lane_id');
   var cell_treatment_id = $(element).attr('cell_treatment_id');
   /* map_key is "cell_treatment.id_[lane.id]" */
   var map_key = $(element).attr('data-map_key');
```

Comment:
Feel free to leave as is but this can also be written as `$(element).data('map_key')`